src/App.tsx

```diff
@@ -1,14 +1,52 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+const getVisibleTodos = (
+  todos: Todo[],
+  sortType: string,
+  query: string,
+): Todo[] => {
+  let filtered = todos;
+
+  if (sortType === 'active') {
```

Comment:
We can create an enum to not depend on strings

Something like:
```suggestion
  if (sortType === SortType.Active) {
```